runtime/sql/sql_test.go

```diff
@@ -19,3 +19,17 @@ func TestSanity(t *testing.T) {
 	err := isolate.Close()
 	require.NoError(t, err)
 }
+
+func TestSanityGetAST(t *testing.T) {
+	isolate := NewIsolate()
+
+	sql := "select 1 as foo, 'hello' as bar"
+	schema := `{ "tables": [] }`
+
+	res := isolate.getAST(sql, schema)
```

Comment:
I would improve sanity tests here in the next PR, it actually requires for Go side to generate proto-based Go code in a build step - so it can be done as a separate PR